packages/paste-website/src/components/site-wrapper/SiteHeader.tsx

```diff
@@ -13,6 +13,18 @@ import {SiteBodyContext} from './SiteBodyContext';
 export const SiteHeader: React.FC = () => {
   const {isPasteTheme} = React.useContext(SiteBodyContext);
   const theme = useTheme();
+  const [value, setValue] = React.useState('');
+
+  React.useEffect(() => {
+    if (typeof window !== `undefined` && window.docsearch != null) {
+      window.docsearch({
```

Comment:
I'd be pretty tempted to put this in the `html.tsx` file as well, to be honest. Just as a `<script>` tag after `postBodyComponents`